vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cmd.go

```diff
@@ -161,6 +161,7 @@ __custom_func() {
    * events (aka 'ev')
    * endpoints (aka 'ep')
    * horizontalpodautoscalers (aka 'hpa')
+   * images
```

Comment:
Either add all the cluster scoped resources or don't add any.  One at a time doesn't make sense.

Also, if you do this, create a reflective test to keep us up to date with types in the scheme.